recipes/rye/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set version = "0.33.0" %}
+
+package:
+  name: rye
+  version: {{ version }}
+
+source:
+  fn: rye-{{ version }}.tar.gz
+  url: https://github.com/astral-sh/rye/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c712eae1d68da599751ff868c77e9cf04eddec8a084e6da5614c3d8bdc267f61
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+    - cmake
+    - make
+
+test:
+#  requires:
+#    - m2-grep  # [win]
+  commands:
+    - rye --version
```

```suggestion
    - rye version
```

https://rye-up.com/guide/commands/version/